components/mdc/Drawer/Drawer.svelte

```diff
@@ -24,6 +28,7 @@ onMount(() => {
 const isMenuItemActive = (currentUrl, menuItemUrl) => currentUrl === menuItemUrl
 
 $: currentUrl = window.location.pathname
+$: toggle, toggleDrawer()
```

Comment:
Does $: toggle, toggleDrawer() work the same as $: toggle && toggleDrawer() ?  At one point I thought so, but I haven't been able to get it to work recently.